lib/services/role.go

```diff
@@ -51,6 +51,7 @@ var AdminUserRules = []Rule{
 	NewRule(KindSession, RO()),
 	NewRule(KindTrustedCluster, RW()),
 	NewRule(KindEvent, RO()),
+	NewRule(KindUser, RO()),
```

Comment:
should `admin` be able to create users too?
I'd expect this to be `RW()`

Reply:
I went with Ben's suggestion to start with `list, read` permissions first: https://github.com/gravitational/teleport/pull/4150#issuecomment-667310648. The full access broke a test: https://github.com/gravitational/teleport/blob/master/lib/auth/tls_test.go#L1310, and I didn't want to change this test until things get finalized.